src/main/java/at/hannibal2/skyhanni/config/features/chat/ChatConfig.java

```diff
@@ -136,4 +136,14 @@ public String toString() {
     @ConfigEditorBoolean
     @FeatureToggle
     public boolean hideSkyMall = true;
+
+    @Expose
+    @ConfigOption(
+        name = "Pet Drop Rarity",
+        desc = "Shows what rarity the pet drop is in the pet drop message.\n" +
+            "§6§lPET DROP! §r§5§lEPIC §r§5Slug §6(§6+1300☘)"
```

Comment:
Those reset characters aren't needed (a color also resets previous color + formatting)
```suggestion
            "§6§lPET DROP! §5§lEPIC §5Slug §6(§6+1300☘)"
```